README.md

```diff
@@ -106,18 +107,19 @@ var options = {}
 
 // auth is required
 // it can either be a username & password...
-var auth = {
+options.auth = {
   username: 'ngoldman',
   password: 'XXXXXXXX'
 }
+
 // or an API token
-var auth = {
+options.auth = {
   token: 'XXXXXXXX'
 }
 
-ghRelease(options, auth, function (err, result) {
+ghRelease(options, function (err, result) {
   if (err) throw err
-  console.log('Release URL: ' + result)
+  console.log(result) // create release response: https://developer.github.com/v3/repos/releases/#response-4
```

Comment:
you are already returning the raw response.

In the cli we fish out the url https://github.com/ngoldman/gh-release/blob/50379e870acb9bc1d6e53a2e898726c7eceff961/bin/cli.js#L103 but this doc is for the Node API and they'll be getting the response in full